src/base/bittorrent/private/resumedatasavingmanager.h

```diff
@@ -41,8 +41,11 @@ class ResumeDataSavingManager : public QObject
     explicit ResumeDataSavingManager(const QString &resumeFolderPath);
 
 public slots:
-    void save(const QString &filename, const QByteArray &data) const;
-    void remove(const QString &filename) const;
+    void save(const QString &filename, const QByteArray &data);
+    void remove(const QString &filename);
```

Comment:
I think you can avoid dropping the `const`: https://stackoverflow.com/questions/5781449/is-it-possible-to-emit-a-qt-signal-from-a-const-method

Reply:
The methods can change state of object so they're not `const` by their essence.